en/sql/schema/table_stmt.rst

```diff
@@ -97,11 +106,16 @@ Use **CREATE TABLE** statement to create a table. A table with the same name as
 *   *table_comment_string*: specifies a table's comment
 *   *column_comment_string*: specifies a column's comment.
 *   *index_comment_string*: specifies an index's comment.
+*   *deduplicate_level*: Specifies the deduplicate level (0 to 14). For details, see :ref:`deduplicate_overview`.
 
 .. note::
 
     *   **DBA** and **DBA** members can create tables in different schemas. If a user is neither **DBA** nor **DBA** member, tables can only be created in the schema of that user.
 
+.. note::
+
+    *deduplicate_level*\ is an integer from 0 to 14. 0 means an index with the same configuration as CUBRID 11.2 or earlier without the **DEDUPLICATE** option.
```

Comment:
```suggestion
    *deduplicate_level*\ is an integer from 0 to 14. 0 means an index with the same configuration as CUBRID 11.2 or earlier which doesn't support the **DEDUPLICATE** option.
```

Reply:
@mhoh3963 
This sentence will be corrected to accommodate hgryoo's comment above.